.changeset/unlucky-hairs-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/lit': minor
+---
+
+Allows using the Constructor for rendering components
```

Comment:
Is this the breaking change from supporting `<my-element>` to only supporting `<MyElement>`? I'd make sure to call this change out much more explicitly.

Reply:
Update: looks like it is. This definitely deserves a thorough explanation!

Reply:
There is that breaking change in this PR, yes. But this specific change enables using the Constructor for LitElements. This was originally supported when I first created the renderer but must have been accidentally broken at some later time.

Fair enough, will add that to this changeset.

Reply:
Updated this to clarify what is going on and what was deprecated.